src/htcondor_es/convert_to_json.py

```diff
@@ -588,23 +589,32 @@
     "FINISHED": "finished",
 }
 
-_launch_time = int(time.time())
+clogger = get_task_logger(__name__)  # Celery logger
```

Comment:
I'm not big fan of python logger as it mask printouts. Here you use yet another logger on top of python logger and I wonder if such complexity is required. Simple print will work just fine, and you can replace logger and clogger calls with single print statement. But I'm not against either, if you think that loggers are better feel free to leave them as is.

Reply:
I changed them to prints. Logger put timestamps to logs. Now it's running in k8s and each log line has its own timestamp (in UTC!) thanks to k8s and we don't need logger.